asset/js/setup.js

```diff
@@ -32,20 +32,18 @@ function setupAnchorsForFixedNavbar() {
   jQuery('#content-wrapper').css('padding-top', `calc(${headerHeight}px)`);
```

Comment:
would it be possible to just insert the padding for the main container and not its 3 sub containers?
![fixedheader](https://user-images.githubusercontent.com/3306138/79070271-48551500-7d07-11ea-8139-fe3940cd0be3.png)

Reply:
As far as I'm concerned it doesn't work...

Reply:
ok, might be better to leave it this way as well: `.css('nav-inner')` needs to be offsetted with the original padding

![navbarpadding](https://user-images.githubusercontent.com/3306138/79241304-7b350f80-7ea5-11ea-8261-90ef036473bf.gif)

Reply:
just did that, wondering if we need to add `1rem` to `content-wrapper` as well, but probably better leave it as a future first timer issue?

Reply:
Good idea, would make it seem less squeezed to the top. Let's leave it as a first timer issue then, though if you want to include it here it'd be nice as well